src/navigator/Navigator.tsx

```diff
@@ -404,6 +405,11 @@ const backupScreens = (Navigator: typeof Stack) => (
       component={AccounSetupFailureScreen}
       options={AccounSetupFailureScreen.navOptions}
     />
+    <Navigator.Screen
+      name={Screens.BackupIntroduction}
```

Comment:
we now have 2 screens by the same name - should i separate them? one is in the drawer navigator and this is in the stack, the navigation seems to work fine like this but not sure if it's strictly the best thing to do 😅

Reply:
😄 should be ok if navigation works as expected.